Jekyll example used in "Containers used in generating this lesson" currently not working

The example used in the "_Containers used in generating this lesson_" episode (`_episodes/07-containers-for-continuous-integration.md`) currently seems to be broken.

When you try to run the example using the command provided:

```
docker run --rm -it -v ${PWD}:/srv/jekyll -p 127.0.0.1:4000:4000 jekyll/jekyll:3.7.3 make serve
```

You get an error:

```
Could not locate Gemfile or .bundle/ directory
make: *** [Makefile:26: serve] Error 10
```

It seems that the `Makefile` is trying to run `bundle exec jekyll serve` which relies on the various gem dependencies being installed by [_bundler_ ](https://bundler.io/) either in a global or local environment. For example, by convention, these dependencies would often be installed in the `vendor/bundle` subdirectory under the project.

Since we're using the jekyll docker container, we should have all the required dependencies installed globally and bundler shouldn't be required.

I was sure this was working originally and it looks like the `jekyll` command in the `Makefile` was changed in commit 4e1e6b02 from `jekyll` to `bundle exec jekyll`.

The options here are:
 - Add an instruction to the lesson requiring the installation of the dependencies before running the `docker` command (e.g. run `bundle install --path vendor/bundle`)
  OR
 - Change the command in the `Makefile` back to `jekyll`, although I'm not sure what unintended consequences this may have

At the same time, I'd recommend increasing the jekyll version to something more recent. For example, we could use 3.9.0 as listed in the [GitHub Pages dependency versions](https://pages.github.com/versions/) as the version currently used.

--- a/_config.yml
+++ b/_config.yml
@@ -102,4 +102,4 @@ exclude:
 # Turn on built-in syntax highlighting.
 highlighter: rouge
 
-remote_theme: carpentries/carpentries-theme
+remote_theme: carpentries/carpentries-theme@main

